src/printer.js

```diff
@@ -857,13 +857,13 @@ function printPathNoParens(path, options, print) {
       const parent = path.getParentNode()
       if (n.shorthand) {
         parts.push(path.call(print, 'value'))
-        if (n.computed) {
+        if (shouldPrintComputedKeyBrackets(n)) {
```

Comment:
Not sure if it should respect the presence of existing `[`/`]` + interpolated string key? Eg should this:
```
["#{a}"]: b
```
get formatted like that or like:
```
"#{a}": b
```
Seems borderline whether that's in Prettier's domain or not